catalog/ui/catalog-ui-search/src/main/webapp/component/singletons/metacard-definitions.js

```diff
@@ -24,6 +24,13 @@ define([
             this.getMetacardTypes();
             this.getDatatypeEnum();
         },
+        isHiddenTypeExceptThumbnail: function(id){
```

Comment:
Why is this not just added to `isHiddenType`?  This way requires someone to know to run both methods.  Are there cases where you want thumbnails to be hidden?